static/app/views/dashboards/widgetBuilder/buildSteps/sortByStep/sortBySelectors.tsx

```diff
@@ -180,7 +180,6 @@ export function SortBySelectors({
           <ArithmeticInput
             name="arithmetic"
             type="text"
-            required
```

Comment:
cc @shruthilayaj this never worked due to https://github.com/getsentry/sentry/blob/a3096e5c1dafb4a2afec2fbe62a5dfae7a1881df/static/app/components/core/input/index.tsx#L91